target/linux/ramips/dts/TOTOLINK-LR1200.dts

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "totolink,lr1200", "mediatek,mt7628dan";
```

Comment:
@981213 `mt7628dan` ? :-) Other then this LGTM.

Reply:
Yeah. That's exactly how this chip is named.